modules/owncloud_web/pages/web_with_oC10.adoc

```diff
@@ -0,0 +1,35 @@
+= ownCloud Web on ownCloud Server 10.x
+
+To ease the transition from ownCloud server 10.x to oCIS, we recommend starting to use the new Web UI. ownCloud users will benefit from the following additions and improvements:
```

Comment:
have a `:description:` and introduction item here too. This should always be present right from the start in every document.